postfix/Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM debian:stretch-slim
 
 LABEL maintainer="Mriyam Tamuli <mbtamuli@gmail.com>"
+LABEL org.label-schema.schema-version="1.0.0-rc2"
```

Comment:
This is not related to EasyEngine version. It has to be rc1 - http://label-schema.org/rc1/

Reply:
Done